Makefile

```diff
@@ -238,6 +240,35 @@ handler-nodejs: processor
 
 IMAGES_TO_PUSH += $(NUCLIO_DOCKER_HANDLER_NODEJS_ALPINE_IMAGE_NAME)
 
+# java
+
+NUCLIO_HANDLER_JAVA_DOCKERFILE_PATH = pkg/processor/build/runtime/java/docker/Dockerfile.handler
+NUCLIO_DOCKER_HANDLER_JAVA_ALPINE_IMAGE_NAME=nuclio/handler-java:$(NUCLIO_DOCKER_IMAGE_TAG_WITH_ARCH)
+
+handler-java: processor
```

Comment:
In a recent PR, Ilay nicely expanded the `-f` and `-t` - please do the same (`--file`, `--tag`)

Reply:
OK